packages/core/src/quote/Quote.ts

```diff
@@ -99,17 +104,36 @@ export async function fromAttesterSignedInput(
 
 export async function createAttesterSignature(
   quoteInput: IQuote,
-  attesterIdentity: IDidDetails,
-  signer: KeystoreSigner
+  attesterIdentity: DidDetails,
+  {
+    signer,
```

Comment:
Yes, the idea is that anything that is domain-specific, like the quote input and the attester, are parameters, and everything that is the same across several operations, like the keystore, the key selection, the resolver in some cases, would be in an object, as to make it more memorable for users to interact with. What do you think? Otherwise we would end up having four to six parameters, whose order might be hard to remember.

Reply:
as discussed, let's just put optional parameters into the object and also assign that to the empty object.